src/util.h

```diff
@@ -8,11 +8,17 @@
 #define SECP256K1_UTIL_H
 
 #include "../include/secp256k1.h"
+#include "checkmem.h"
 
+#include <string.h>
 #include <stdlib.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <limits.h>
+#if defined(_MSC_VER)
+// For SecureZeroMemory
```

Comment:
nit: `//`-style comments not allowed in C89. Just for consistency, it appears that MSVC doesn't care...